src/reachy2_sdk/parts/arm.py

```diff
@@ -287,6 +287,52 @@ def inverse_kinematics(
             answer = np.round(answer, round).tolist()
         return answer
 
+    def get_default_pose_matrix(self, common_pose: str = "default") -> npt.NDArray[np.float64]:
+        """Return the 4x4 pose matrix of default robot poses."""
+        if common_pose not in ["default", "elbow_90"]:
+            raise ValueError(f"common_pose {common_pose} not supported! Should be 'default' or 'elbow_90'")
+
+        if common_pose == "elbow_90":
+            elbow_pitch = -90
+        else:
+            elbow_pitch = 0
+
+        if self._part_id.name == "r_arm":
+            pose = self.forward_kinematics([0, -15, -15, elbow_pitch, 0, 0, 0])
+        else:
+            pose = self.forward_kinematics([0, 15, 15, elbow_pitch, 0, 0, 0])
+
+        return pose
+
+    def get_pose_matrix(
```

Comment:
Is it for arms only? It seems that a generic function that builds a pose. It can go to utils right?